tools/sonarcloud/README.md

```diff
@@ -17,6 +17,30 @@ The following Pon maintained quality profiles are available:
 
 - [Javascript](https://github.com/pondevelopment/pon-developer-guide/blob/main/tools/sonarcloud/sonarcloud_quality_profile_javascript.xml)
 
+### Code Coverage Reporting
+SonarCloud supports [reporting of test coverage](https://docs.sonarcloud.io/enriching/test-coverage/overview/) information. This enables teams and Management to get more insight into the code coverage of their project.
+
+To set this up, the steps are as follows:
+
+ 1. Switch SonarCloud configuration to CI based analysis (instead of automatic analysis).
+ 2. Update CI workflow to generate test coverage report.
+ 3. Add or update `sonar-project.properties` file adding field `sonar.javascript.lcov.reportPaths` that points to the location of the coverage report.
+ 4. Configure a CI workflow to generate the code coverage report for `main` or `master` and feature branches.
+
+
+### Analysis Scope
```

Comment:
Can we add some examples regarding pricing, like: the game is pon was 10k lines - Magento is 300k so developers can give a ballpark price?